doc/mediasdk-man.md

```diff
@@ -4139,16 +4139,16 @@ The application can attach this extended buffer to the [mfxVideoParam](#mfxVideo
 `GPB` | Turn this option OFF to make HEVC encoder use regular P-frames instead of GPB.<br><br>See the [CodingOptionValue](#CodingOptionValue) enumerator for values of this option
 `LowDelayHrd` | Corresponds to AVC syntax element low_delay_hrd_flag (VUI).<br><br>See the [CodingOptionValue](#CodingOptionValue) enumerator for values of this option.
 `MotionVectorsOverPicBoundaries` | When set to OFF, no sample outside the picture boundaries and no sample at a fractional sample position for which the sample value is derived using one or more samples outside the picture boundaries is used for inter prediction of any sample.<br><br>When set to ON, one or more samples outside picture boundaries may be used in inter prediction.<br><br>See the [CodingOptionValue](#CodingOptionValue) enumerator for values of this option.
-`MaxFrameSizeI` | Same as [mfxExtCodingOption2](#mfxExtCodingOption2)`::MaxFrameSize` but affects only I-frames. Maximum allowed frame size per profile/level will be calculated in driver and be used when MaxFrameSizeI and MaxFrameSizeP are both set to 0. MaxFrameSizeI must be set if MaxFrameSizeP is set.
-`MaxFrameSizeP` | Same as [mfxExtCodingOption2](#mfxExtCodingOption2)`::MaxFrameSize` but affects only P/B-frames. If MaxFrameSizeP equals 0, MaxFrameSizeI will be used for all frame types. Maximum allowed frame size per profile/level will be calculated in driver and be used when MaxFrameSizeI and MaxFrameSizeP are both set to 0.
+`MaxFrameSizeI` | Same as [mfxExtCodingOption2](#mfxExtCodingOption2)`::MaxFrameSize` but affects only I-frames. MaxFrameSizeI must be set if MaxFrameSizeP is set. If MaxFrameSizeI and MaxFrameSizeP are not specified or greater than spec limitation, spec limitation will be applied to the values.
+`MaxFrameSizeP` | Same as [mfxExtCodingOption2](#mfxExtCodingOption2)`::MaxFrameSize` but affects only P/B-frames. If MaxFrameSizeP equals 0, MaxFrameSizeI will be used for all frame types. If MaxFrameSizeI and MaxFrameSizeP are not specified or greater than spec limitation, spec limitation will be applied to the values.
```

Comment:
I would formulate like this "If MaxFrameSizeP equals 0, the SDK sets MaxFrameSizeP equal to MaxFrameSizeI. If MaxFrameSizeP is not set or greater than spec limitation, spec limitation will be applied to the sizes of P/B frames".